include/cglm/affine-mat.h

```diff
@@ -152,7 +152,7 @@ glm_inv_tr(mat4 mat) {
   glm_inv_tr_sse2(mat);
 #else
   CGLM_ALIGN_MAT mat3 r;
-  CGLM_ALIGN(16) vec3 t;
+  CGLM_ALIGN(8) vec3 t;
```

Comment:
can you add 1 space before **vec3** please, I'm obsessive about code styling 🤔

Reply:
ok! no problem. Pushed now.